src/views/v2/tickets/item/quantity/add.php

```diff
@@ -14,8 +14,8 @@
  *
  * @version TBD
  *
- * @var $this Tribe__Tickets__Editor__Template
- * @var Tribe__Tickets__Ticket_Object $ticket The ticket object.
+ * @var	Tribe__Tickets__Editor__Template	$this Template object.
```

Comment:
* :no_entry_sign: Spaces must be used for mid-line alignment; tabs are not allowed
* :no_entry_sign: Spaces must be used for mid-line alignment; tabs are not allowed

<sub>hash: 4ea2fdd366b2528983aec025f3e7a003</sub>

Reply:
Avoid using the tab key in these places, some editors will tab with spaces when in this context but it's possible certain editors will use tab characters and this will come up.

Reply:
```suggestion
 * @var	Tribe__Tickets__Editor__Template $this   Template object.
```